package.json

```diff
@@ -102,59 +102,65 @@
       }
     ]
   },
+  "resolutions": {
+    "long": "5.2.0"
+  },
   "dependencies": {
-    "@noble/secp256k1": "^1.5.2",
-    "@xmtp/proto": "^3.34.0",
-    "@xmtp/user-preferences-bindings-wasm": "^0.3.5",
-    "async-mutex": "^0.4.0",
-    "elliptic": "^6.5.4",
-    "ethers": "^5.5.3",
-    "js-sha3": "^0.9.3",
-    "long": "^5.2.0"
+    "@noble/secp256k1": "1.7.1",
+    "@xmtp/proto": "3.34.0",
+    "@xmtp/user-preferences-bindings-wasm": "0.3.6",
+    "async-mutex": "0.4.0",
+    "elliptic": "6.5.4",
+    "ethers": "5.5.3",
```

Comment:
I worry about these exact matches for common deps causing dependency bloat for integrators who use slightly different versions of common libraries like `ethers`

Reply:
this is a temporary measure to prevent issues when switching from NPM to yarn. once this PR merges, i'll start upgrading the dependencies with proper versions.